Preprocessing/ErrorLogger.py

```diff
@@ -0,0 +1,17 @@
+class ErrorLogger:
+    def __init__(self, logger):
+        self.logger = logger
+        self.errors = []
+
```

Comment:
Myślę że 

- dodać `self.__is_finalized = False`
- `finalize` ustawia to na true
- dodać dtor, który rzuci exceptiona i wypisze errory, jeżeli nie sfinalizowano loggera

taki sanity check, bo możę być krzywe gówno jak tak sobie zignorujesz errory